### PR TITLE
selenium-standalone: fix driver keys condition

### PR DESCRIPTION
**Pull request by mgrybyk**
## Proposed changes

Fix drivers keys condition. There should be at least one driver provided, not two

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: packages/wdio-selenium-standalone-service/src/launcher.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ export default class SeleniumStandaloneLauncher {
         this.skipSeleniumInstall = Boolean(options.skipSeleniumInstall)
 
         // simplified mode
-        if (options.drivers && Object.keys(options.drivers).length > 1) {
+        if (options.drivers && Object.keys(options.drivers).length > 0) {
```
Comment (christian-bromann) [inline packages/wdio-selenium-standalone-service/src/launcher.ts:56]: Maybe we move `0` to a const to name it property which helps to understand its meaning.

**Review by christian-bromann — COMMENTED** [review]
Do you think it makes sense to add a unit test for this?

File: packages/wdio-selenium-standalone-service/src/launcher.ts (hunk 1 above)
Comment (mgrybyk) [reply packages/wdio-selenium-standalone-service/src/launcher.ts:56]: I'll add a test and add function with meaningful name

**Comment by mgrybyk** [comment]
Let's release this one

**Review by christian-bromann — APPROVED** [review]
👍

**Pull request merged**